components/speedreader/buildflags.gni

```diff
@@ -1,5 +1,5 @@
 import("//build/config/features.gni")
 
 declare_args() {
-  enable_speedreader = !is_ios
+  enable_speedreader = !is_android
```

Comment:
don't we want to disable it for both ios and android currently?

Reply:
i believe ios is based on a different codebase...

Reply:
but we do use the flag in various other components, and the CI seems to build some libraries used by ios - there still doesn't seem to be a need to build it for ios, so should be disabled?

Reply:
I think components for ios are manually selected, so we shouldn't explicitly disable - it is never reached. @bsclifton please correct me if I'm wrong?